Date: 2011-08-03 18:36:44+0200
From: Jan Engelhardt <jengelh@inai.de>
Upstream: sent

Always do tesselation, because otherwise, map hacks like
self-referencing sectors (cf.
http://doom.wikia.com/wiki/Making_a_self-referencing_sector) will not
be rendered.

---
 src/gl_main.c |    3 +++
 1 file changed, 3 insertions(+)

Index: prboom-plus-2.5.1.4/src/gl_main.c
===================================================================
--- prboom-plus-2.5.1.4.orig/src/gl_main.c
+++ prboom-plus-2.5.1.4/src/gl_main.c
@@ -36,6 +36,9 @@
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
+#ifndef USE_GLU_TESS
+#	define USE_GLU_TESS 1
+#endif
 
 #include "gl_opengl.h"
 
